discovery-infra/test_infra/config/_cluster_config.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import dataclass
+
+from test_infra.config import EnvConfig
+from test_infra.helper_classes.cluster_config import ClusterConfig, ClusterInstallConfig
+
+
+@dataclass
+class DefaultInstallConfig(ClusterInstallConfig):
```

Comment:
Not sure it is very clear for me why we need DefaultInstallConfig and ClusterInstallConfig ? 
I think that we will always will use default and will change only some values in it in case test needs it.
Why not to have same functionality in ClusterInstallConfig and delete this class?